cpp/src/gandiva/lru_cache.h

```diff
@@ -100,11 +136,58 @@ class LruCache {
     }
   }
 
+  arrow::util::optional<value_type> getObject(const key_type& key) {
```

Comment:
Ditto

Reply:
I checked, and I might be the case, so I capitalized the first letter.